include/constants/move.h

```diff
@@ -23,47 +23,47 @@ enum MoveFlag2
 enum AccuracyType
 {
     // Accuracy used for all moves.
-    ACCURACY_TYPE_GLOBAL,
+    ACCURACY_1,
```

Comment:
Feel like we are losing some descriptiveness here for both `ACCURACY_1` and `ACCURACY_2`

Reply:
The global/offensive split isn't entirely accurate. There are a number of offensive moves, mainly fixed-damage moves, that don't use accuracy 2.